.jenkins/JenkinsfileTestLinuxRelease

```diff
@@ -110,19 +110,15 @@ def ACCTestOeRelease(String label, String version) {
                             make run
                         done
                        """
-                dcap.ContainerRun("${DOCKER_REGISTRY}/oetools-full-${version}:latest", "clang-8", task, "--cap-add=SYS_PTRACE --device /dev/sgx:/dev/sgx")
+                dcap.ContainerRun("${DOCKER_REGISTRY}/oetools-full-${version}:latest", "clang-10", task, "--cap-add=SYS_PTRACE --device /dev/sgx:/dev/sgx")
             }
         }
     }
 }
 
-parallel "ACC1604 SGX1-FLC Container RelWithDebInfo" : { ACCContainerTest('ACC-1604', '16.04') },
-         "ACC1804 SGX1-FLC Container RelWithDebInfo" : { ACCContainerTest('ACC-1804', '18.04') },
-         "ACC1604 SGX1-FLC clang-8 Debug" :            { ACCTest('ACC-1604', '16.04', 'clang-8', 'Debug') },
-         "ACC1604 SGX1-FLC clang-8 Release" :          { ACCTest('ACC-1604', '16.04', 'clang-8', 'Release') },
-         "ACC1604 SGX1-FLC clang-8 RelWithDebInfo" :   { ACCTest('ACC-1604', '16.04', 'clang-8', 'RelWithDebInfo') },
-         "ACC1804 SGX1-FLC clang-8 Debug" :            { ACCTest('ACC-1804', '18.04', 'clang-8', 'Debug') },
-         "ACC1804 SGX1-FLC clang-8 Release" :          { ACCTest('ACC-1804', '18.04', 'clang-8', 'Release') },
-         "ACC1804 SGX1-FLC clang-8 RelWithDebInfo" :   { ACCTest('ACC-1804', '18.04', 'clang-8', 'RelWithDebInfo') },
+parallel "ACC1804 SGX1-FLC Container RelWithDebInfo" : { ACCContainerTest('ACC-1804', '18.04') },
+         "ACC1804 SGX1-FLC clang-10 Debug" :            { ACCTest('ACC-1804', '18.04', 'clang-10', 'Debug') },
+         "ACC1804 SGX1-FLC clang-10 Release" :          { ACCTest('ACC-1804', '18.04', 'clang-10', 'Release') },
```

Comment:
`Release` build type is not recommended, and `RelWithDebInfo` or `Debug` should be used